Makefile

```diff
@@ -266,7 +266,12 @@ clean:
 	rm -f *.cover.out
 	rm -f coverage.txt
 
+dep:
+	dep ensure -v
+	rm -rfv ./vendor/github.com/ethereum/go-ethereum/crypto/secp256k1/libsecp256k1/
+	cp -av "$${GOPATH}/src/github.com/ethereum/go-ethereum/crypto/secp256k1/libsecp256k1/" ./vendor/github.com/ethereum/go-ethereum/crypto/secp256k1/libsecp256k1/
```

Comment:
Why we need do this?

Reply:
The dep program will treat the libsecp256k1 c package as a unused golang package and remove it on prune stage. However if we disable prune for the ethereum dependency, whole ethereum source code will be merged into CovenantSQL. So a manually code sync is required.